wallet-new/integration/AccountSpecs.hs

```diff
@@ -91,11 +106,14 @@ accountSpecs wRef wc =
                     , pmtSpendingPassword = Nothing
                     }
             amounts <- pick $ shuffle [1..5]
+
             let addrAndAmount = zip (map (\(addr : _) -> addr) $ map accAddresses accs) amounts
-            forM_  addrAndAmount $ \(addr, amount) ->
+
+            etxnsResp <- forM addrAndAmount $ \(addr, amount) -> do
+                -- we have to wait until the previous transaction is accommodated in the blockchain
+                liftIO $ threadDelay 120000000
                 run $ postTransaction wc (payment amount addr)
 
-            liftIO $ threadDelay 90000000
```

Comment:
By making a small change to the helper below:

```hs
    pollTransaction :: V1 Txp.TxId -> IO ()
    pollTransaction tid = do
        resp <- getTransactionIndex wc Nothing Nothing Nothing >>= shouldPrismFlipped _Right
        let Just tx = find ((== tid) . txId) (wrData resp)
        if (txStatus tx `elem` [InNewestBlocks, Persisted]) then
            return ()
        else
            threadDelay (5 * oneSecond) >> pollTransaction tid
```

We can just write this:

```hs
            run $ forM_ addrAndAmount \(addr, amount) -> do
                etxn <- postTransaction wc (payment amount addr)
                txn  <- fmap wrData etxn `shouldPrism` _Right
                ("waiting for Tx to be accepted", pollTransaction (txId txn)) `noLongerThan` (120 * oneSecond)
```

Which would greatly speed up the test.